x-pack/plugins/fleet/common/types/models/package_policy.ts

```diff
@@ -78,7 +78,9 @@ export interface NewPackagePolicy {
   namespace?: string;
   enabled: boolean;
   is_managed?: boolean;
-  policy_id: string;
+  // deprecated
+  policy_id?: string;
```

Comment:
Making `policy_id` optional caused the most type check failures across plugins. I think we have to make it optional as going forward we want to support multiple `policy_ids`.